libs/create-langchain-integration/template/package.json

```diff
@@ -22,7 +22,7 @@
     "lint:fix": "yarn lint --fix",
     "clean": "rm -rf dist/ && NODE_OPTIONS=--max-old-space-size=4096 node scripts/create-entrypoints.js pre",
     "prepack": "yarn build",
-    "release": "release-it --only-version --config .release-it.json",
+    "release": "release-it --ci --config .release-it.json",
```

Comment:
What's the change here for?

Reply:
Actually, i need to delete these scripts since the code now runs `release-it` and passes in args all inside a `spawn` in the script.

Reply:
Ah but I drop `--only-version` since it now explicitly passes the version number in. And `--ci` just says say yes to all the questions it asks, eg `Publish to npm?`. As if it was running in an async ci environment